plugins/importexport/medra/MedraExportPlugin.inc.php

```diff
@@ -102,11 +103,25 @@ function depositXML($objects, $context, $filename) {
 		assert(is_readable($filename));
 		$xml = file_get_contents($filename);
 		assert($xml !== false && !empty($xml));
-
+		
+		// Select the language
+		$language = 'en_US';
+		foreach($objects as $object) {
+		    $language = $object->getLocale();
+		}
+		$lang = 'eng';
+		if(substr($language, 0, 2) == 'it'){
+		    $lang = 'ita';
+		} else if(substr($language, 0, 2) == 'de'){
+		    $lang = 'ger';
+		}
+		
 		// Instantiate the mEDRA web service wrapper.
 		$ws = new MedraWebservice($endpoint, $username, $password);
 		// Register the XML with mEDRA.
-		$result = $ws->upload($xml);
+		//$result = $ws->upload($xml);
```

Comment:
(Dead code should be removed)

Reply:
done.

Reply:
Hi Alec, could you let me know about the adjustement on code?